### DISCUSSION TITLE
Add rortation to the tp command

### DISCUSSION THREAD

**Discussion by Josh65-2201** [EssentialsX/Essentials#4931] [Ideas and Feature Suggestions] [answered]
Minecraft allows x y z yaw pitch but essential doesnt witch is quite annoying when people arent used to adding minecraft: before the command

**Comment by Chew** [comment, marked as answer]
`/tppos` supports all those arguments already, in that order.

```
/tppos <x> <y> <z> [yaw] [pitch] [world]
```

**Reply by Josh65-2201** [reply]
not when using /tp witch is what i would expect it to work on

**Reply by pop4959** [reply]
Essentials uses different tp commands so it is expected. `/tp` is for player teleportation and `/tppos` is for coordinates/position.
